pkgs/development/tools/misc/coreboot-toolchain/default.nix

```
@@ -17,12 +17,12 @@ let
 
     stdenvNoCC.mkDerivation rec {
       pname = "coreboot-toolchain-${arch}";
-      version = "4.16";
+      version = "4.17";
 
       src = fetchgit {
         url = "https://review.coreboot.org/coreboot";
-        rev = version;
-        sha256 = "073n8yid3v0l9wgwnrdqrlgzaj9mnhs33a007dgr7xq3z0iw3i52";
+        rev = "8da4bfe5b573f395057fbfb5a9d99b376e25c2a4";
```

```suggestion
        rev = version;
```
https://review.coreboot.org/plugins/gitiles/coreboot/+/refs/tags/4.17